kubernetes/provider.go

```diff
@@ -124,6 +124,7 @@ func Provider() terraform.ResourceProvider {
 			"kubernetes_service_account":           resourceKubernetesServiceAccount(),
 			"kubernetes_storage_class":             resourceKubernetesStorageClass(),
 			"kubernetes_cluster_role_binding":      resourceKubernetesClusterRoleBinding(),
+			"kubernetes_stateful_set":              resourceKubernetesStatefulSet(),
```

Comment:
not a big deal, but this list should probably be in alphabetical order? vscode can do this for you, as other editors.

Reply:
Makes total sense. I'll make sure they're sorted.
This could be a good candidate for some fancy custom linting rules.